test/test-server/kafka_rest_router.go

```diff
@@ -708,6 +715,34 @@ func handleKafkaRestLink(t *testing.T) http.HandlerFunc {
 					LinkState:       cckafkarestv3.PtrString("AVAILABLE"),
 				})
 				require.NoError(t, err)
+			} else if link == "link-5" {
+				includeTasks := r.URL.Query().Get("include_tasks")
+				var tasks []cckafkarestv3.LinkTask
+				if includeTasks == "true" {
+					tasks = []cckafkarestv3.LinkTask{
+						*cckafkarestv3.NewLinkTask("ConsumerOffsetSync", "ACTIVE", []cckafkarestv3.LinkTaskError{}),
+						*cckafkarestv3.NewLinkTask("AutoCreateMirror", "NOT_CONFIGURED", []cckafkarestv3.LinkTaskError{}),
+						*cckafkarestv3.NewLinkTask("AclSync", "IN_ERROR", []cckafkarestv3.LinkTaskError{
```

Comment:
It might be better to initialize the struct directly here instead of calling a `New()` function and immediately dereferencing it (which is a bit weird)

Reply:
Ok.

Reply:
This is done.